### DISCUSSION TITLE
Sry this is my explanation

### DISCUSSION THREAD

**Discussion by sillymoon** [quiple/galmuri#50] [General]
How can I support you specifically?

_Originally posted by @quiple in https://github.com/quiple/galmuri/issues/46#issuecomment-1872553473_
            sry my bad, maybe there's something wrong with my expression or understanding. I mean I have made a font under OFL1.1 and have taken some reference from the hangul part in *Galmuri*, i will write who made this hangul part in the end. Sry to bother

**Comment by quiple** [comment]
Nice!
